ddtrace/llmobs/_evaluators/ragas/faithfulness.py

```diff
@@ -0,0 +1,36 @@
+import math
+import time
+from typing import Optional
+
+
+class RagasFaithfulnessEvaluator:
```

Comment:
Once we introduce more evaluators we'll also introduce a base evaluator class, but there's no need for that right now.